lib/analytics/reddit-analyzer.js

```diff
@@ -0,0 +1,72 @@
+"use strict";
+
+function analyze(redditJson) {
+  var i;
+  var stats = {
+    subreddit: {},
+    author: {},
+    domain: {},
+    ngram: {},
+    is_self: {},
+    gilded: 0,
+    num_comments: 0,
+    score: 0
+  };
+
+  if (redditJson.data && redditJson.data.children) {
```

Comment:
Some more comments for your if statements and for loops would be nice, though I do understand what they're doing :)